cli/src/clients/datafusion_helpers.rs

```diff
@@ -627,7 +627,7 @@ pub async fn get_locked_keys_status(
                 sis.last_attempt_deployment_id,
                 sis.next_retry_at,
                 sis.last_start_at
-            FROM sys_status ss
+            FROM sys_invocation_status ss
             LEFT JOIN sys_invocation_state sis ON ss.id = sis.id
```

Comment:
I have to admit that I find `sys_invocation_status` and `sys_invocation_state` a bit too close in Levenshtein distance. I always find myself thinking about what each table represents. Maybe as a follow-up, can we try to find more distinct names?

Reply:
Same here, we'll have to fix this at a later point.